wrappers/src/shared_realm_cs.cpp

```diff
@@ -94,6 +94,25 @@ class TestHelper {
         auto transaction = Realm::Internal::get_transaction_ref(*realm);
         return Realm::Internal::get_db(*realm)->has_changed(transaction);
     }
+
+    static DB::version_type get_latest_version(const SharedRealm& realm)
+    {
+        auto transaction = Realm::Internal::get_transaction_ref(*realm);
+        return transaction->get_version_of_latest_snapshot();
+    }
+
+    static DB::version_type get_current_version(const SharedRealm& realm)
+    {
+        auto transaction = Realm::Internal::get_transaction_ref(*realm);
+        return transaction->get_version();
+    }
+
+    static void notify_realm(const SharedRealm& realm)
+    {
+        auto& cord = Realm::Internal::get_coordinator(*realm);
+        cord.on_change();
+    }
```

Comment:
This looks very hacky - I think we should reach out to the core team and find a more future-proof approach.